static/js/main.js

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
Can we rename this js file to `members-download.js`? Since it's only used for the download stuff. `main.js` is normally for the shared code.

Reply:
@yuanzhou  The idea is that currently we have a bunch of js directly embedded into various html files. I was thinking in the future we have a bunch of separate modules that when compiled will just be a main.js file. But for now, I guess I can rename it because that functionality is not there.